newIDE/app/src/ProjectCreation/ProjectPreCreationDialog.js

```diff
@@ -24,6 +24,7 @@ type Props = {|
   onClose: () => void,
   onCreate: ProjectCreationSettings => Promise<void>,
   sourceExampleName?: string,
+  preferences?: any,
```

Comment:
The type is `Preferences` but you can entirely remove this prop (see my other comment)